### ISSUE TITLE
Algorithmic debugger is broken

### ISSUE THREAD

**Issue by bilderbuchi** [OpenModelica/OpenModelica#12923]
I'm trying to debug a crashing (not failing, crashing) model, and tried to use the algorithmic debugger. Turns out it does not work at all. 

Open `Modelica.Blocks.Examples.Filter` and press simulate with algorithmic debugger
Observe some compiler warnings (#3972)
Find out there is no way to start the debugger, the console output tab says:
`Following error has occurred <b>Process failed to start: The system cannot find the file specified.</b> GDB arguments are <b>"-q -nw -i mi --args C:/Users/myuser/AppData/Local/Temp/OpenModelica/OMEdit/Modelica.Blocks.Examples.Filter/Filter.exe"</b>`
I checked and that exe file _is_ at that location.

Windows 10, OpenModelica v1.25.0-dev-31-ge9e3dc1832

@sjoelund maybe you know what's broken here?

**Comment by adeas31** [comment]
Check the gdb options `Tools|Options|Algorithmic Debugger` https://openmodelica.org/doc/OpenModelicaUsersGuide/latest/omedit.html#debugger-options

**Comment by bilderbuchi** [comment]
> Is gdb installed?

Yes, the OM Command Prompt shows version 14.1 from C:\OpenModelica1.25.0-dev-64bit\tools\msys\ucrt64\bin\gdb.exe



> Check the gdb options `Tools|Options|Algorithmic Debugger` https://openmodelica.org/doc/OpenModelicaUsersGuide/latest/omedit.html#debugger-options

What do you want to know?
![image](https://github.com/user-attachments/assets/151fe057-2ecf-49e9-bc49-48fa0bd10351)

**Comment by sjoelund** [comment]
There is a missing `/` between `ucrt64` and `bin`

**Comment by adeas31** [comment]
The path to gdb is wrong. There is a missing slash in the default path. I will fix it.

**Comment by casella** [comment]
Ouch, I guess we should also fix this in 1.24.0 😅 

Thanks @bilderbuchi for reporting. It's difficult to automate tests of this kind of issues, particularly considering that we are compiling OMEdit for, like, 15 different OS platforms.

**Comment by bilderbuchi** [comment]
> The path to gdb is wrong. There is a missing slash in the default path. I will fix it.

Ah, I did not spot that! Confirmed, hardcoding the fixed path makes the debugger work again. Thanks for the prompt analysis!

> It's difficult to automate tests of this kind of issues, particularly considering that we are compiling OMEdit for, like, 15 different OS platforms.

Sure, although "just" doing one Linux + Windows + MacOS already covers a lot of ground ^^
In this case what weighs in more is that it's hard to test GUI-related things in an automated fashion.

**Comment by casella** [comment]
> Sure, although "just" doing one Linux + Windows + MacOS already covers a lot of ground ^^
> In this case what weighs in more is that it's hard to test GUI-related things in an automated fashion.

Sure. In fact, it's the combination of these two issues which is deadly.

Community feedback is another way to handle that 😃

**Comment by adeas31** [comment]
Fixed with #12924 and ported to maintenance/v1.24 with #12928.

@bilderbuchi don't forget to remove the hard coded path otherwise you will have the same issue with new version.
